test/unit/Pagination.test.js

```diff
@@ -31,201 +25,172 @@ describe('Pagination', () => {
     let component;
 
     before(() => {
-      component = shallow(<Pagination hits={40} />);
+      component = shallow(<Pagination total={40} />);
     });
 
-    it('should be wrapped in a pagination wrapper', () => {
-      expect(component.hasClass('pagination')).to.be.true;
+    it('should be wrapped in a .nypl-results-pagination nav wrapper', () => {
+      expect(component.hasClass('nypl-results-pagination')).to.equal(true);
+      expect(component.type()).to.equal('nav');
     });
 
-    it('should not have any buttons since the results are less than 51', () => {
-      expect(component.find('button')).to.have.length(0);
+    it('should not have any Links since the results are less than 51', () => {
+      expect(component.find('Link').children().length).to.equal(0);
     });
 
     it('should display how many items are displayed', () => {
-      expect(component.find('span').text()).to.equal('1 - 40 of 40');
+      expect(component.find('.page-count').text()).to.equal('Page 1 of 1');
     });
 
     it('should have a descriptive aria-label', () => {
       expect(component.find('span').prop('aria-label'))
-        .to.equal('Displaying 1 - 40 out of 40 total items.');
+        .to.equal('Displaying page 1 out of 1 total pages.');
     });
   });
 
   describe('With 400 results on the default page', () => {
     let component;
 
     before(() => {
-      component = mount(<Pagination hits={400} />);
+      component = mount(<Pagination total={400} />);
+    });
+
+    it('should have a first page class for the text display', () => {
+      expect(component.find('.first').length).to.equal(1);
     });
 
-    it('should have a "next page" button since there are more than 51 items', () => {
-      expect(component.find('button')).to.have.length(1);
-      expect(component.find('.next').text()).to.equal('Next Page');
+    // The SVG titles should maybe not be here:
+    it('should have a "next" link since there are more than 51 items', () => {
+      expect(component.find('Link').children().length).to.equal(1);
+      expect(component.find('.next-link').text()).to.equal('Wedge Right Arrow Next');
```

Comment:
Could check there's no previous page link. Nice to have but no necessary I think.

Reply:
True, but it's already checking that there's only one Link component which is the Next link.

Reply:
But added one.

Reply:
OK!